### PR TITLE
fix: fixes around formatting numbers and arrange edit order dialog

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #[Issue number here]

# Description ℹ️

- [x] normalize numbers (remove trailing 0s and fix wrong numbers in market info liquidity)
- [x] increase edit order col span dialog
- [x] rename "Propose a change to this market" as the current link does not direct you to a market-specific form

image
image

### CODE REVIEW

File: libs/market-info/src/components/market-info/info-key-value-table.tsx [hunk 1]
```diff
@@ -40,8 +40,9 @@ const Row = ({
       return value;
     }
     if (decimalPlaces) {
-      return `${parseFloat(
```
Comment (MadalinaRaicu) [inline libs/market-info/src/components/market-info/info-key-value-table.tsx:43]: this breaks market-info target stake and supplied stake values.

Comment (MadalinaRaicu) [reply libs/market-info/src/components/market-info/info-key-value-table.tsx:43]: NOTE: I will handle the follow up on ALL number formatters :D https://github.com/vegaprotocol/frontend-monorepo/issues/2179

File: apps/trading/components/select-market/select-market-columns.tsx [hunk 2]
```diff
@@ -554,7 +554,14 @@ export const columnsPositionMarkets = (
     {
       kind: ColumnKind.Position,
       value: (
-        <p className={signedNumberCssClass(openVolume || '')}>{openVolume}</p>
+        // TODO add decimals to open volume
```
Comment (mattrussell36) [inline apps/trading/components/select-market/select-market-columns.tsx:557]: chore (blocking): this comment seems irrelevant now

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: apps/trading/components/select-market/select-market-columns.tsx (hunk 2 above)
Comment (MadalinaRaicu) [reply apps/trading/components/select-market/select-market-columns.tsx]: removed.

**Review by mattrussell36 — APPROVED** [review]
